app/core/settings.py

```diff
@@ -39,5 +40,11 @@ class Settings(BaseSettings):
     MONGODB_URI: str = "mongodb://root:example@localhost:27017"
     MESSAGEQUEUE_URI: str = "amqp://guest:guest@localhost:5672"
 
+    AUTH_ALGORITHM = "HS256"
+    AUTH_SECRET_KEY = "09d25e094faa6ca2556c818166b7a9563b93f7099f6f0f4caa6cf63b88e8d3e7"
```

Comment:
This doesn't seem like this should be here.

Reply:
This and other private setting will need to be moved before we deploy.